servo-tidy.toml

```diff
@@ -32,15 +33,12 @@ rand = [
 [ignore]
 # Ignored packages with duplicated versions
 packages = [
-  "core-graphics",
   "crossbeam-deque",
   "crossbeam-epoch",
   "crossbeam-utils",
   "gl_generator",
```

Comment:
A few duplicate crates were introduced in #22202. Those can probably be removed now.

Reply:
Yup, forgot about them :p Will remove them once I'm done with the other PR I'm rebasing